web/src/main/java/pl/isa/freshmenindustries/game/GameController.java

```diff
@@ -21,6 +21,16 @@ public GameController(GameService gameService) {
         this.gameService = gameService;
     }
 
+    @GetMapping("start-new-game")
+    public String startNewGame(Model model,
+                               @ModelAttribute("response") Response response,
+                               @ModelAttribute("game") Game game) {
+        model.addAttribute("gamesList", gameService.getAllGames().getData())
```

Comment:
Jeżeli getAllGames() zwraca Response, to fajnie byłoby dodać obsługę błędów, które ten Response może zwrócić. Gdy dostaliśmy poprawne dane, to dodajemy listę, która ukrywa się w getData(), a jak nie, to fajnie byłoby dodać obsługę błędów i wyświetlanie jakiegoś powiadomienia użytkownikowi.